Task1:print my information, teachers information, solution task

danakik opened the pull request.

danakik committed:
print name lesson and number task
print my name,surname and number record book
print name teachers
print solution task

vloooo commented on task_1.py:
```diff
@@ -0,0 +1,10 @@
+num_3 = 127.24 - 21.1
+num_4 = 3.2
+num_5 = ((num_1 * num_2) / num_3) - num_4
+print(round(num_5, 5))
```
You need to add blank line at the end of the file

vloooo commented on task_1.py:
```diff
@@ -0,0 +1,10 @@
+num_2 += 2.2
+num_3 = 127.24 - 21.1
+num_4 = 3.2
+num_5 = ((num_1 * num_2) / num_3) - num_4
```
It's a good idea to divide the calculation for several lines. but the implementation looks strange for me.
Frankly, now I cannot  provide you an ultimate solution. 
let's discuss, how it could be arranged better at the lab.

vloooo commented on task_1.py:
```diff
@@ -0,0 +1,10 @@
+print("Multiparadigm programming languages : Task 1")
+print("Danil Mashkin, FI-9119")
+print("Vladyslav Valeriyovych, "*44,end = "Vladyslav Valeriyovych. \n" )
```
good try. extraordinary decision.
I would suggest to change it a bit in such way. 
s = "Vladyslav Valeriyovych, " * 45
print(f'{s[:-2]}\n')

danakik replied:
Thanks for the advice
but if so then
print(f'{s[:-1]}\n')

vloooo replied:
ok, maybe. 
I didn't run)))

vloooo replied:
BTW, you missed some spaces.

danakik committed:
print name lesson and number task
print my name,surname and number record book
print name teachers
print solution task

vloooo commented on task_1.py:
```diff
@@ -0,0 +1,8 @@
+print('Multiparadigm programming languages : Task 1')
+print("Danil Mashkin, FI-9119")
+
+teacher_45 = "Vladyslav Valeriyovych, " * 45
```
single quotes ;)
